invenio_rdm_records/records/api.py

```diff
@@ -23,10 +23,15 @@ class BibliographicRecord(Record):
     model_cls = RecordMetadata
 
     index = IndexField(
-        'rdmrecords-records-record-v1.0.0', search_alias='rdmrecords-records')
+        "rdmrecords-records-record-v1.0.0", search_alias="rdmrecords-records"
```

Comment:
minor: I am in favour of consistency, but I am not sure if this sort of "styling" changes are good for history tracking :)